src/NuGet.Core/NuGet.Protocol/Providers/ServiceIndexResourceV3Provider.cs

```diff
@@ -141,6 +141,12 @@ public ServiceIndexResourceV3Provider()
                             log,
                             token);
                     }
+                    catch (OperationCanceledException ex)
+                    {
+                        var message = ExceptionUtilities.DisplayMessage(ex);
+                        log.LogInformation(message);
+                        throw;
```

Comment:
Given that this is in NuGet.Protocol, can we maybe update the docs?

Reply:
Sure, can you point to the docs you're referring?

Reply:
Re-reading it, I realized I didn't provide enoguh detail :D I was referring the xml docs in for the codepath in question.

Reply:
I'll add comments with whatever behavior results from addressing PR feedback.

Reply:
Comments good?